app/modules/conversations/conversations_router.py

```
@@ -118,6 +120,75 @@ async def post_message(
+                        logger.info(
+                            f"DEBUG: Uploaded image {image.filename} with ID: {upload_result.id}"
+                        )
+                    except Exception as e:
+                        logger.error(
+                            f"DEBUG: Failed to upload image {image.filename}: {str(e)}"
+                        )
+                        # Clean up any successfully uploaded attachments
+                        for uploaded_id in attachment_ids:
+                            try:
+                                await media_service.delete_attachment(uploaded_id)
+                            except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)